pom.xml

```diff
@@ -860,7 +860,6 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-test</artifactId>
 			<version>${spring.version}</version>
-			<scope>test</scope>
```

Comment:
Is that by accident?

Reply:
No. We can not write `src/main/java` customizing code against `WebTestClient` if `WebTestClient` is test-scoped.

Reply:
Ah, that's unfortunate. We should make the dependency optional then because otherwise we always leak the test JAR into users production classpaths.

Reply:
Will do.